modules/rostests/apitests/ntdll/NtAdjustPrivilegesToken.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * PROJECT:         ReactOS API tests
+ * LICENSE:         GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:         Tests for the NtAdjustPrivilegesToken API
+ * COPYRIGHT:       Copyright 2021 George Bișoc <george.bisoc@reactos.org>
+ */
+
+#include "precomp.h"
+
+static
+LUID
+ConvertPrivLongToLuid(
```

Comment:
Seems to be a candidate for a simple macro 🤔

Reply:
I disagree -- if anything, it's a complicated macro. What advantage does the macro give you?
The function gives you type safety, not having to worry about multiline-macro-do-while stuff, and correct syntax highlighting.